src/strategies/staked-defi-balance/index.ts

```diff
@@ -15,40 +13,69 @@ export async function strategy(
   options,
   snapshot
```

Comment:
Hi @taha-abbasi you should use `snapshot` to calculate voting power at the proposal block, else voting power will be unpredictable and can cause lot of issues on the proposal

Reply:
@ChaituVR thanks for the suggestions, so the strategy is using `snapshot` just not the `snapshot` at the root of examples.json. I left it there to pass the tests as I cannot modify the tests.

You can see snapshot being used here:
```typescript
for (const params of options) {
    const paramNetwork = params.network.toString();
    const paramSnapshot =
      typeof params.snapshot === 'number' ? params.snapshot : 'latest';
``` 
This also allows "latest" as a value which is what will be utilized most of the time. The `snapshot` will be different per network defined in params, hence the root `snapshot` value is not suitable.